Nodejs/Product/Nodejs/Nodejs.csproj

```diff
@@ -901,8 +901,9 @@
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
       <IncludeInVSIX>true</IncludeInVSIX>
     </Content>
-    <Content Include="Debugger\RemoteDebug\RemoteDebug.js">
+    <Content Include="RemoteDebug\RemoteDebug.js">
```

Comment:
This makes more sense to exist in Debugger\*, so would prefer to set the VSIXPath than change the location on disk.